subsys/caf/modules/buttons.c

```diff
@@ -70,8 +90,15 @@ static int set_cols(uint32_t mask)
 						       col[i].pin, val);
 			}
 		} else {
-			err = gpio_pin_configure(gpio_devs[col[i].port],
-						 col[i].pin, GPIO_INPUT);
+			gpio_flags_t flags = GPIO_INPUT;
+			gpio_flags_t pull = (IS_ENABLED(CONFIG_CAF_BUTTONS_POLARITY_INVERSED) ?
+					    (GPIO_PULL_UP) : (GPIO_PULL_DOWN));
+
+			if (enforce_state) {
+				flags |= pull;
+			}
```

Comment:
I think that an additional comment on why it is necessary to use pull up or pull-downs would help to clarify the intent of this code

Reply:
bump

Reply:
I was thinking that comment is no longer needed after removal of `enforce_state`. Anyway, added back